geopmpy/geopmpy/launcher.py

```diff
@@ -141,7 +141,7 @@ def __init__(self, argv):
         parser.add_argument('--geopm-debug-attach', dest='debug_attach', type=str)
         parser.add_argument('--geopm-preload', dest='preload', action='store_true', default=False)
         parser.add_argument('--geopm-hyperthreads-disable', dest='allow_ht_pinning', action='store_false', default=True)
-        parser.add_argument('--geopm-ompt-disable', dest='ompt_disable', action='store_true', default=False)
+        parser.add_argument('--geopm-ompt-enable', dest='ompt_enable', action='store_true', default=False)
```

Comment:
Could consider guiding users with existing scripts that use the old option. E.g., 
```
ompt_group = parser.add_mutually_exclusive_group()
ompt_group.add_argument(enable...)
ompt_group.add_argument(disable..., help="Deprecated since OMPT regions are disabled by default.")
...
if opts.ompt_disable:
    print('Warning: --geopm-ompt-disable is deprecated since OMPT regions are now disabled by default. The option may be removed in future releases.', file=sys.stderr)
    # don't abort since ignoring this flag achieves that user's desired behavior anyways
    # or don't even warn, and treat "user explicitly asked for the default" as normal behavior
```

Reply:
I've added in a check in geopmlaunch and also in Environment.cpp